src/check_cache.cc

```diff
@@ -97,12 +105,12 @@ Status CheckCache::CacheResponse(const Attributes& attributes,
   return Status::OK;
 }
 
+// TODO: need to hook up a timer to call Flush.
```

Comment:
In the latest mixer API, there's is an 'expiration' field in CheckResponse. This is intended to tell the proxy how long it can hang on to the results of the Check call. The mixer client should annotate every cache entry with this expiration time. If at the time of check, current time > expiration time then remove the cache entry and call mixer. And have a background flusher that grovels through everything and kills any entry passed its expiration time.